cra/sample/pr-pipeline.yaml

```diff
@@ -32,6 +32,14 @@ spec:
     - name: ibmcloud-api
       description: the ibmcloud api
       default: https://cloud.ibm.com
+    - name: tf-dir
```

Comment:
Existing pipelines using the current master branch of this tekton repositories will be broken as the the `tf-dir` does not have a default value so it will require the environment property of the already created cd tekton pipeline using this definition to be changed.
I see it as a breaking change

Reply:
I have provided the default value: "".